.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,29 @@
+name: auth-landing CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm ci
+      - name: Format code
+        run: npm run format
```

Comment:
On CI, we should run `prettier --list-difference` instead. That command exits with 1 if any file is formatted. Maybe we can add `npm run check-format` to alias that command.

Reply:
👍